src/test/resources/org/prebid/server/it/openrtb2/openx/test-auction-openx-request.json

```diff
@@ -19,6 +19,13 @@
               "bar2"
             ]
           }
+        },
+        "gpid": "gpidvalue",
```

Comment:
We are trying to test different data in unit tests, not in integration
Pls remove this